os/dev/nullradio.c

```diff
@@ -27,7 +27,6 @@ transmit(unsigned short transmit_len)
 static int
 send(const void *payload, unsigned short payload_len)
 {
-  prepare(payload, payload_len);
   return transmit(payload_len);
```

Comment:
If the call to `prepare` is removed because it does nothing, should not the call to `transmit` also be removed for the same reason?

Reply:
Fixed.